notification/conf/bootstrap.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+/var/lib/dpkg/info/ca-certificates-java.postinst configure
```

Comment:
You're right, I did that here: https://github.com/guardian/mobile-platform/pull/197. I'll remove that line